### ISSUE TITLE
Maui support on linux

### ISSUE THREAD

**Issue by webczat** [dotnet/maui#20011]
Hello,
I would like maui to add linux support.
Note I don't see any open issue for this, that's why I opened a new one, but if it's not true, please feel free to close.

**Comment by Symbai** [comment]
> I don't see any open issue for thi

Because this is not an issue. Its under "Discussions" with over 800 comments: https://github.com/dotnet/maui/discussions/339

**Comment by webczat** [comment]
Oh... nice

**Comment by mattleibow** [comment]
Duplicate of https://github.com/dotnet/maui/discussions/339

**Comment by NicTanghe** [comment]
TLDR: It's definitely an issue, and it should stay open even if no action is taken.

Me and everyone who mains Linux highly disagrees with the " this is a discussion " answer.

Intentionally moving the issue into a space where people will disagree with each other over nothing and go off-topic instead of trying to get everything thought out and working.  Is an understandable move.

Although, you can not in good faith say that it is not an issue as well.

In my opinion, either the old thread should be cleaned and changed into an issue or this one should be opened for people who have practical ideas about where problems are located and/or how to solve these problems to get things working. 
As well as people asking for help on the subject.

That being said, it will probably take a long time until action is actually taken on the subject. So it will stay open for a while.

PLZ ignore all improper grammar and/or spelling as those are unrelated.
